core/cont/inc/TList.h

```diff
@@ -77,43 +77,43 @@ friend  class TListIter;
 
    TList() : fAscending(kTRUE) { }
    TList(TObject *) : fAscending(kTRUE) { } // for backward compatibility, don't use
-   virtual           ~TList();
-   virtual void      Clear(Option_t *option="");
-   virtual void      Delete(Option_t *option="");
-   virtual TObject  *FindObject(const char *name) const;
-   virtual TObject  *FindObject(const TObject *obj) const;
-   virtual TIterator *MakeIterator(Bool_t dir = kIterForward) const;
-
-   virtual void      Add(TObject *obj) { AddLast(obj); }
+   ~TList() override;
+   void       Clear(Option_t *option = "") override;
+   void       Delete(Option_t *option = "") override;
+   TObject *  FindObject(const char *name) const override;
+   TObject *  FindObject(const TObject *obj) const override;
+   TIterator *MakeIterator(Bool_t dir = kIterForward) const override;
+
+   void              Add(TObject *obj) override { AddLast(obj); }
    virtual void      Add(TObject *obj, Option_t *opt) { AddLast(obj, opt); }
-   virtual void      AddFirst(TObject *obj);
+   void              AddFirst(TObject *obj) override;
    virtual void      AddFirst(TObject *obj, Option_t *opt);
-   virtual void      AddLast(TObject *obj);
+   void              AddLast(TObject *obj) override;
    virtual void      AddLast(TObject *obj, Option_t *opt);
-   virtual void      AddAt(TObject *obj, Int_t idx);
-   virtual void      AddAfter(const TObject *after, TObject *obj);
+   void              AddAt(TObject *obj, Int_t idx) override;
+   void              AddAfter(const TObject *after, TObject *obj) override;
    virtual void      AddAfter(TObjLink *after, TObject *obj);
-   virtual void      AddBefore(const TObject *before, TObject *obj);
+   void              AddBefore(const TObject *before, TObject *obj) override;
    virtual void      AddBefore(TObjLink *before, TObject *obj);
-   virtual TObject  *Remove(TObject *obj);
+   TObject *         Remove(TObject *obj) override;
    virtual TObject  *Remove(TObjLink *lnk);
            TObject  *Remove(const TObjLinkPtr_t &lnk) { return Remove(lnk.get()); }
-   virtual void      RemoveLast();
-   virtual void      RecursiveRemove(TObject *obj);
-
-   virtual TObject  *At(Int_t idx) const;
-   virtual TObject  *After(const TObject *obj) const;
-   virtual TObject  *Before(const TObject *obj) const;
-   virtual TObject  *First() const;
-   virtual TObjLink *FirstLink() const { return fFirst.get(); }
-   virtual TObject **GetObjectRef(const TObject *obj) const;
-   virtual TObject  *Last() const;
-   virtual TObjLink *LastLink() const { return fLast.get(); }
-
-   virtual void      Sort(Bool_t order = kSortAscending);
-   Bool_t            IsAscending() { return fAscending; }
-
-   ClassDef(TList,5)  //Doubly linked list
+           void      RemoveLast() override;
+           void      RecursiveRemove(TObject *obj) override;
+
+           TObject *         At(Int_t idx) const override;
```

Comment:
I suppose this is a deficiency in clang-format, the start column is off by quite a bit.

Reply:
It seems you are right :/ I fixed it by hand.